kasa/cli/main.py

```diff
@@ -252,9 +253,7 @@ async def cli(
     if target != DEFAULT_TARGET and host:
         error("--target is not a valid option for single host discovery")
 
-    logging_config: dict[str, Any] = {
-        "level": logging.DEBUG if debug > 0 else logging.INFO
-    }
+    logging_config: dict[str, Any] = {"level": logging.WARNING}
```

Comment:
Is this to suppress upstream warnings for the onvif lib? If yes, I think we should have a custom config for it and not for our own package.

Reply:
Fixed